recipes/beyla-golden-signals/beyla-daemonset.yaml

```diff
@@ -33,6 +33,29 @@ spec:
     spec:
       serviceAccountName: beyla
       hostPID: true
+      initContainers:
+        - name: mount-bpf-fs
+          image: grafana/beyla:1.5.0
+          args:
+          # Create the directory using the Pod UID, and mount the BPF filesystem.
+          - 'mkdir -p /sys/fs/bpf/$BEYLA_BPF_FS_PATH && mount -t bpf bpf /sys/fs/bpf/$BEYLA_BPF_FS_PATH'
```

Comment:
Do you remember why the AppArmor annotation was needed? Since we are now creating the bpf mount ourselves, I wonder if we could create it in a location or with permissions that don't require `unconfined`.

Reply:
https://github.com/GoogleCloudPlatform/opentelemetry-operator-sample/pull/96#issuecomment-2070008420